### PR TITLE
Add check for semver-compliant version string

### PR DESCRIPTION
**Pull request by rasenplanscher**
Piggy-backs on #37 and should be reviewed and merged after that.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -146,9 +151,17 @@ function runVersionCommand(command, callback) {
     const commandDescription = JSON.stringify(command);
 
     if (!execError) {
-      return callback(null, {
-        version: stdout,
-      });
+      const version = stdout.trim();
+
+      if (semver.valid(version)) {
```
Comment (parshap) [inline index.js]: 👍

**Comment by parshap** [comment]
This will need a rebase -- @rasenplanscher mind doing that or giving me [permission to push](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to this branch?

**Comment by rasenplanscher** [comment]
@parshap done 👍

**Comment by parshap** [comment]
Woops, looks like I forgot to merge and release this before. Done now. Released as v4.0.3.

**Pull request merged**